war/src/main/scss/components/_section.scss

```diff
@@ -3,7 +3,6 @@
 .jenkins-section {
   border-top: 2px solid var(--panel-border-color);
   padding: var(--section-padding) 0 0 0;
-  max-width: 1800px;
```

Comment:
Simplified the styling, let's just fill the available width and let the parent container control how wide that can be.